core/src/components/button/test/theme-ionic/button.e2e.ts

```diff
@@ -0,0 +1,82 @@
+import { expect } from '@playwright/test';
+import type { E2EPage } from '@utils/test/playwright';
+import { configs, test } from '@utils/test/playwright';
+
+class ButtonFixture {
+  readonly page: E2EPage;
+  readonly screenshotFn?: (file: string) => string;
+
+  constructor(page: E2EPage, screenshot?: (file: string) => string) {
+    this.page = page;
+    this.screenshotFn = screenshot;
+  }
+
+  async selectOption(select: string, optIndex: number) {
+    const selectElem = this.page.locator(`ion-select#select-${select}`);
+    await selectElem.click();
+
+    const selectOptElem = this.page.locator(`ion-alert .alert-radio-group button:nth-child(${optIndex + 1})`);
+    await selectOptElem.click();
+
+    const selectOkBtn = this.page.locator(`ion-alert .alert-button-group button:nth-child(2)`);
+    await selectOkBtn.click();
+  }
+
+  async checkScreenshot(modifier: string) {
+    const { screenshotFn } = this;
+
+    if (!screenshotFn) {
+      throw new Error(
+        'A screenshot function is required to take a screenshot. Pass one in when creating ButtonFixture.'
+      );
+    }
+
+    await expect(this.page).toHaveScreenshot(screenshotFn(`${modifier}-button`));
```

Comment:
Capturing an entire page screenshot will result in additional diffs in the future (from changes not specific to the button) and significantly larger file sizes. 

Can we either:
1. Capture just the individual elements (smallest file size with this approach)
2. Capture a screenshot of the "Preview" and "States" containers?

This will remove the "Preview Options" section and header from being captured in the screenshot.

Reply:
Done!